main.go

```diff
@@ -109,7 +109,6 @@ func main() {
 
 	utils.RunSH("dependencies", "mv luet /usr/bin/luet && mkdir -p /etc/luet/repos.conf.d/")
 	utils.RunSH("dependencies", "curl -L https://raw.githubusercontent.com/mocaccinoOS/repository-index/master/packages/luet.yml --output /etc/luet/repos.conf.d/luet.yml")
```

Comment:
I'm not sure if we should keep this line either. This point to really old version of packages. Why would we need this at all? Do we install anything else other than luet?

Reply:
there are other plugins available, probably not used